sway-core/Cargo.toml

```diff
@@ -31,7 +32,9 @@ sha2 = "0.9"
 smallvec = "1.7"
 sway-ir = { version = "0.6.1", path = "../sway-ir" }
 sway-types = { version = "0.6.1", path = "../sway-types" }
+sway-utils = { version = "0.6.1", path = "../sway-utils" }
 generational-arena = "0.2"
+bigint = "4.3.3"
```

Comment:
Did you deliberately add these?

Reply:
Oups - I needed that for something else and I don't anymore. I'll take it out.